### PR TITLE
Fix: Add missing arguments for SSHBox in evaluation

### PR DESCRIPTION
**Pull request by SmartManoj**
Related to #3041

Update `evaluation/webarena/run_infer.py` to help SSH connection issues and missing arguments in `DockerSSHBox` initialization.

* Update `DockerSSHBox` initialization in Line 35 with required arguments:
  * `config=config.sandbox`
  * `persist_sandbox=False`
  * `workspace_mount_path=config.workspace_mount_path`
  * `sandbox_workspace_dir=config.workspace_mount_path_in_sandbox`
  * `cache_dir=config.cache_dir`
  * `run_as_devin=config.run_as_devin`

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/OpenDevin/OpenDevin/issues/3041?shareId=c6657d88-c9a7-4492-a427-b7b634a3f9f8).

### CODE REVIEW

File: evaluation/webarena/run_infer.py [hunk 1]
```diff
@@ -32,7 +32,14 @@
 def get_sandbox():
     global docker_ssh_box
     if docker_ssh_box is None:
-        docker_ssh_box = DockerSSHBox()
```
Comment (SmartManoj) [inline evaluation/webarena/run_infer.py:35]: This should throw an error. right?

**Review by xingyaoww — APPROVED** [review]

**Comment by xingyaoww** [comment]
@SmartManoj can we add these for all the benchmark though?

**Comment by SmartManoj** [comment]
Need to pass sandbox here?

https://github.com/OpenDevin/OpenDevin/blob/547c510848d0b4782ec2d023375469f734f64a6c/evaluation/EDA/run_infer.py#L121-L130

**Comment by tobitege** [comment]
> Need to pass sandbox here?
> 
> https://github.com/OpenDevin/OpenDevin/blob/547c510848d0b4782ec2d023375469f734f64a6c/evaluation/EDA/run_infer.py#L121-L130

EDA doesn't use any sandbox, so no change there.

**Pull request merged**